src/components/common/TextBlockNarrow/index.tsx

```diff
@@ -0,0 +1,18 @@
+import type { BaseBlock } from '@/components/Home/types'
+import layoutCss from '@/components/common/styles.module.css'
+import { Container, Typography } from '@mui/material'
+import clsx from 'clsx'
+import css from './styles.module.css'
+
+type TextBlockNarrowProps = BaseBlock & { id?: string };
+
+const TextBlockNarrow = ({ title, text, id }: TextBlockNarrowProps) => {
+  return (
+    <Container className={clsx(layoutCss.containerTiny, css.block)} id={id}>
+      <Typography variant="h3">{title}</Typography>
+      {text}
+    </Container>
+  )
```

Comment:
Looking at Figma, the block should sit inside <Container/> 
```suggestion
    <Container className={layoutCss.containerTiny} id={id}>
      <div className={css.block}>
        <Typography variant="h3">{title}</Typography>
        {text}
      </div>
    </Container>
  )
```